packages/fhir-location-management/src/components/LocationForm/utils.ts

```diff
@@ -67,7 +67,7 @@ export const getLocationFormFields = (
   parentId?: string
 ): LocationFormFields => {
   return {
-    ...location,
+    ...(location ?? defaultFormField),
```

Comment:
This spreading doesn't seem like it works? we should maybe move to explicit ternary `...(location ? defaultFormField : {}),`